integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImageCluster.java

```
@@ -0,0 +1,410 @@
+        customDirPath.toString(), "clusteraiReplace.txt");
+
+    // create a plain text file
+    Path fileInCustomDir = Paths.get(customDirPath.toString(), "clusteraiReplace.txt");
+    String fileContent = "patch the domain to replace";
+    assertDoesNotThrow(() -> Files.write(fileInCustomDir, fileContent.getBytes()),
+        "Can't write to file " + fileInCustomDir);
+
+    // create image with plain text file clusteraiReplace.txt
+    logger.info("Create auxiliary image: {0}", miiAuxiliaryImage);
+    createAuxiliaryImage(customAiPath.toString(),
+        Paths.get(RESOURCE_DIR, "auxiliaryimage", "Dockerfile").toString(), miiAuxiliaryImage);
+
+    // push image to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", miiAuxiliaryImage, DOMAIN_IMAGES_REPO);
+      assertTrue(dockerPush(miiAuxiliaryImage), String.format("docker push failed for image %s", miiAuxiliaryImage));
+    }
+
+    // patch a domain using auxiliary image to replace an existing cluster configuration
+    logger.info("Patch domain to replace an existing cluster config to cluster {0}", clusterName);
+    patchDomainClusterWithAuxImageAndVerify(domainUid, domainNamespace, managedServerPrefix, replicaCount,
+        clusterIndex, auxiliaryImageVolumeName, miiAuxiliaryImage, replaceAuxImageLoc, "replace");
+
+    // verify that plain text file only copied to servers in the cluster but not copied to admin server
+    logger.info("Verify that plain text file only copied to servers in the cluster but not copied to admin server");
+    verifyFileInPod(Paths.get(auxiliaryImagePath, customDir, fileInCustomDir.getFileName().toString()).toString(),
+        fileContent);
+  }
+
+  /**
+   * A negative test to verify that model files in an auxiliary images at the cluster scope will be ignored.
+   * The test creates an Aux image containing JMS model files and use it to config the cluster scope configuration.
```

Aux image --> Auxiliary Image

---

Will there be any event/log message when the  Auxiliary Image @ cluster scope contains WLS resource ? Or it will be silently ignored ?

---

testPatchDomainToAddModelsToClusterConfig_neg() --> testPatchDomainToAddModelsToClusterConfigIgnored()